js/game-page.js

```diff
@@ -218,6 +219,7 @@ H5P.ArithmeticQuiz.GamePage = (function ($, UI, ArithmeticType, QuestionsGenerat
 
     this.$scoreWidget = $('<div>', {
       'class': 'h5p-baq-score-widget',
+      'tabindex': '0',
```

Comment:
I don't think we need tabindex for score and several of the other elements that has been given a tabindex, there is no action here so keyboard users does not need to access it.
Readspeakers should be able to access the DOM element even if it doesn't have a tabindex.